.github/workflows/CI.yml

```diff
@@ -366,151 +347,32 @@ jobs:
-    runs-on: ubuntu-20.04
-    env:
-      BRANCH: ${{ needs.prepare_ci_run.outputs.BRANCH }}
-      VERSION: ${{ needs.prepare_ci_run.outputs.VERSION }}
-      KEPTN_SPEC_VERSION: ${{ needs.prepare_ci_run.outputs.KEPTN_SPEC_VERSION }}
-      DATETIME: ${{ needs.prepare_ci_run.outputs.DATE }}${{ needs.prepare_ci_run.outputs.TIME }}
-      GIT_SHA: ${{ needs.prepare_ci_run.outputs.GIT_SHA }}
-    steps:
-      - name: Checkout Code
-        uses: actions/checkout@v2.4.0
-
-      - name: Build Helm Charts
-        id: build_helm_charts
-        run: |
-          if [[ "$BRANCH" == "master" ]] || [[ "$GITHUB_EVENT_NAME" == "pull_request" ]]; then
-            # use VERSION.DATETIME for the image tag (e.g., nightly build)
-            ./gh-actions-scripts/build_helm_charts.sh "${VERSION}" "${VERSION}.${DATETIME}" "${KEPTN_SPEC_VERSION}"
-          else
-            # just use VERSION for the image tag
-            ./gh-actions-scripts/build_helm_charts.sh "${VERSION}" "${VERSION}" "${KEPTN_SPEC_VERSION}"
-          fi
-
-      - name: Upload Helm Chart as an artifact
-        id: upload_helm_chart
-        uses: actions/upload-artifact@v2
-        with:
-          name: keptn-installer
-          path: keptn-charts/*.tgz
+    uses: keptn/keptn/.github/workflows/build-helm-charts.yml@chore/release-automation
+    with:
+      branch: ${{ needs.prepare_ci_run.outputs.BRANCH }}
+      version: ${{ needs.prepare_ci_run.outputs.VERSION }}
+      keptn-spec-version: ${{ needs.prepare_ci_run.outputs.KEPTN_SPEC_VERSION }}
+      datetime: ${{ needs.prepare_ci_run.outputs.DATE }}${{ needs.prepare_ci_run.outputs.TIME }}
 
   ############################################################################
   # Build Docker Images                                                      #
   ############################################################################
-  prepare_github_pr_comment:
-    needs: [prepare_ci_run, unit-tests-microservices]
-    name: Prepare GitHub Report Comment
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Create PR comment
-        if: (github.event_name == 'pull_request') && (github.event.pull_request.head.repo.full_name == github.repository)
-        # only run if this is not a fork - see https://github.com/marocchino/sticky-pull-request-comment/issues/227
-        # note: GH does not allow to access secrets for PRs from a forked repositories due to security reasons
-        # that's fine, but it means we can't comment on the PR in this case
-        uses: marocchino/sticky-pull-request-comment@v2.2.0
-        continue-on-error: true
-        with:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          message: "Docker Build Report"
-          recreate: true
-
   docker_build:
-    needs: [prepare_ci_run, unit-tests-microservices, prepare_github_pr_comment]
     name: Docker Build
+    needs: [prepare_ci_run, unit-tests-microservices]
     runs-on: ubuntu-20.04
```

Comment:
does it miss `uses: keptn/keptn/.github/workflows/build-docker-images.yml@master` ?

Reply:
good catch! i totally overlooked actually making use of the docker build pipeline in the separate file

Reply:
change of plans: i won't make use of the separate pipeline for building docker images in this PR but i will create a follow-up where this will be done. for now, i will remove the docker build pipeline since it's not used in this PR